docs/src/app/components/Divider/ExampleList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Divider from 'material-ui/lib/divider';
+import List from 'material-ui/lib/lists/list';
+import ListItem from 'material-ui/lib/lists/list-item';
+
+import MobileTearSheet from '../mobile-tear-sheet';
+
+const DividerExampleList = () => (
```

Comment:
Also, let me know know if you're okay with this or if you want me to switch it back...

Since this component isn't doing anything with props I changed it like this:

```jsx
/* BEFORE */
const Component = () => {
  return (
    <div>
      <p>Hello world!</p>
    </div>
  );
}
/* After */
const Component = () => (
  <div>
    <p>Hello world!</p>
  </div>
);